client/checkout/woopay/express-button/express-checkout-iframe.js

```diff
@@ -96,6 +96,9 @@ export const expressCheckoutIframe = async ( api, context, emailSelector ) => {
 			buildAjaxURL( getConfig( 'wcAjaxUrl' ), 'get_woopay_session' ),
 			{
 				_ajax_nonce: getConfig( 'woopaySessionNonce' ),
+				order_id: getConfig( 'order_id' ),
```

Comment:
getConfig will always return null for this and following two instances as we don't have them defined in https://github.com/Automattic/woocommerce-payments/blob/9fb0cb4b3dbd48e35cb00fda5976c9d8e39313da/includes/class-wc-payments-checkout.php#L159

Because these three values are always null, preload request validation in WooPay fails.